online-store/domain/src/main/java/com/issoft/auto/domain/Category.java

```diff
@@ -0,0 +1,48 @@
+package com.issoft.auto.domain;
+
+import java.util.List;
+
+public class Category {
+
+    private String name;
+    private List<Product> products;
+    private int numberOfProducts;
+    private String productName;
```

Comment:
what for do you need this param?

Reply:
I removed unused params. will push with new brach
It was created for my first attempt, but after changing the code so that the app works, it became unnecessary